docs/source-2.0/aws/aws-endpoints-region.rst

```diff
@@ -269,9 +269,7 @@ Trait value
       - Description
     * - endpointPatternType
       - ``string``
-      - **Required** The pattern type to use for the partition endpoint.  This value can be set to ``service_dnsSuffix`` to
-        use the ``https://{service}.{dnsSuffix}`` pattern or ``service_region_dnsSuffix`` to use
-        ``https://{service}.{region}.{dnsSuffix}``.
+      - **Required** The pattern type to use for the partition endpoint.  This value should be set to ``aws_recommended`` in almost all cases.
```

Comment:
Should we still require this to be set?  We could default to using recommended

Reply:
Yeah I don't have a strong preference. The only thing I had in mind was that it might lead to a small bug downstream in some code that expects it to always be nonnull if we miss changing it

Reply:
Yeah - I dont have a strong preference either.  Its more explicit to require it and the documentation here and on the trait itself makes it clear what to use.